modules/administration-guide/pages/installing-che-on-minikube.adoc

```diff
@@ -49,3 +49,8 @@ $ {prod-cli} server:status
 ----
 $ {prod-cli} dashboard:open
 ----
+ifeval::["{prod-id-short}" == "che"]
```

Comment:
IMO, you don't need this statement here because this page is upstream-only and does not appear in the downstream docs at all:
```suggestion
```